src/utils/orders.ts

```diff
@@ -25,3 +43,24 @@ export function isOrderUnlimited(amount1: BN | BigNumber | string, amount2: BN |
   const unlimitedAmount = amountToString(UNLIMITED_ORDER_AMOUNT)
   return amountToString(amount1) === unlimitedAmount || amountToString(amount2) === unlimitedAmount
 }
+
+export function toPlaceValidFromOrdersParams(orders: OrderParams[]) {
```

Comment:
We usually specify function return type. TS didn't complain in this case?

Reply:
Apparently not, but I'll add it good point...
not strict enough? maybe is inferring it?